Content/posts/launching-language-and-platform-package-compatibility.md

```diff
@@ -41,32 +43,36 @@ We then decided on a list of Swift compiler versions we’d like to check compat
 * Swift 5.2 (5.2.4)
 * Swift 5.3 (beta)
 
-That’s a lot of builds! That adds up to 40 builds per package, but that’s not all. What if there’s a stable release and a beta release? The stable version might support Swift 4.2 and higher, and the new beta might drop support for anything less than Swift 5.2. Finally, as we also track the status of the default branch, we build for that too. Very quickly, we’ve arrived at a place where we might need to run 120 builds *per package*! With almost 3,200 packages in the index, that’s a *lot* of potential builds - about 380,000! 😅
+That adds up to 40 builds per package, but that’s just the beginning. What if there’s a stable release and a beta release? The stable version might support Swift 4.2 and higher, and the new beta might drop support for anything less than Swift 5.2. Finally, as we also track the status of the default branch, we build for that too. Very quickly, we’ve arrived at a place where we might need to run 120 builds *per package*! With almost 3,200 packages in the index, that’s a *lot* of potential builds - about 380,000! 😅
```

Comment:
It's 32 per package and per version (ARM is only 5.3)

Reply:
Yea I struggled with how to do this. I mention ARM only being 5.3 later.

Reply:
I'll reword it, just had an idea on how.

Reply:
Oh, you mention that in the next paragraph actually.

That's perhaps a place to trim then:

> The number of combinations is less that than the expected 40, because Apple Silicon builds require Swift 5.3. But there's more: What if there’s a stable release and a beta release? The stable version might support Swift 4.2 and higher, and the new beta might drop support for anything less than Swift 5.2. Finally, as we also track the status of the default branch, we build for that too. Very quickly, we’ve arrived at a place where we might need to run 96 builds per package!

The vast majority of packages only have a default branch and release version and thus the number of builds for the full index is "only" around 200,000 😅

Reply:
Pushed a fix for this.